src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.queue.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
```

Comment:
Oh okay my bad, I'm not pushing it to the array, this is called straight away from the `.push` function. The bellow scenario is to prevent the weird case if the library being initiated more than one time.